src/__tests__/ApiContainer.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from '../redux/Store';
+import Apicontainer from '../components/ApiContainer';
+
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest that you add unit tests for pure functions in Redux code.

Read [here](https://www.browserstack.com/guide/unit-testing-of-react-apps-using-jest) for more details